src/features/profile/berths/BerthsContainer.tsx

```diff
@@ -9,16 +9,30 @@ import { getCustomerBerthsProps } from './__fixtures__/mockData';
 import { BERTHS } from './__generated__/BERTHS';
 import {
   DELETE_BERTH_APPLICATION,
-  DELETE_BERTH_APPLICATIONVariables as DELETE_BERTH_APPLICATION_VAR,
+  DELETE_BERTH_APPLICATIONVariables as DELETE_BERTH_APPLICATION_VARS,
 } from './__generated__/DELETE_BERTH_APPLICATION';
 import {
   EXTEND_BERTH_APPLICATION,
-  EXTEND_BERTH_APPLICATIONVariables as EXTEND_BERTH_APPLICATION_VAR,
+  EXTEND_BERTH_APPLICATIONVariables as EXTEND_BERTH_APPLICATION_VARS,
 } from './__generated__/EXTEND_BERTH_APPLICATION';
 import Berths from './Berths';
-import { BERTHS_QUERY, DELETE_BERTH_APPLICATION_MUTATION, EXTEND_BERTH_APPLICATION_MUTATION } from './queries';
-import { getChoicesFromBerthApplication } from './utils';
+import {
+  BERTHS_QUERY,
+  DELETE_BERTH_APPLICATION_MUTATION,
+  EXTEND_BERTH_APPLICATION_MUTATION,
+  UPDATE_BERTH_APPLICATION_MUTATION,
+} from './queries';
+import { getChoiceFromHarborNode, getChoicesFromBerthApplication } from './utils';
 import { BerthApplicationNodeCertainly } from './types';
+import {
+  UPDATE_BERTH_APPLICATION,
+  UPDATE_BERTH_APPLICATIONVariables as UPDATE_BERTH_APPLICATION_VARS,
+} from './__generated__/UPDATE_BERTH_APPLICATION';
+import { UpdateBerthApplicationInput } from '../../../__generated__/globalTypes';
+import { HarborsQuery } from '../../__generated__/HarborsQuery';
+import { OWN_BOATS } from '../boats/__generated__/OWN_BOATS';
+import { HARBORS_QUERY } from '../../queries';
+import { OWN_BOATS_QUERY } from '../boats/queries';
```

Comment:
Similar query in https://github.com/City-of-Helsinki/berth-reservations-ui/blob/master/src/features/queries.ts#L327